renovate.json5

```diff
@@ -0,0 +1,63 @@
+{
+  // This configuration extends the following presets. See
+  // https://docs.renovatebot.com/key-concepts/presets/ for more details.
+  "extends": [
+    "config:recommended",
+  ],
+
+  // Labels to add to all PRs
+  "addLabels": [
+    "dependencies",
+    "renovate",
+  ],
+
+  // Let Renovate update dependencies in branches, without PR,
+  // for updates which have automerge=true.
+  "automergeType": "branch",
+
+  // Show CVEs that have no fixes in the dependency dashboard issue
+  "dependencyDashboardOSVVulnerabilitySummary": "unresolved",
+
+  // Relevant to Major version updates
+  "major": {
+    // Major updates need manual approval from the dependency dashboard issue
+    // before PRs are created.
+    "dependencyDashboardApproval": true,
+  },
+
+  // Fix vulnerabilities known to https://osv.dev/
+  "osvVulnerabilityAlerts": true,
+
+  "packageRules": [
+    // Merge certain updates automatically
+    {
+      "automerge": true,
+      "matchUpdateTypes": [
+        "minor",
+        "patch",
+        "pin",
+        "digest",
+      ]
+    },
+    // Add golang label where applicable
+    {
+      "addLabels": [
+        "golang",
+      ],
+      "matchCategories": [
+        "golang",
+      ]
+    },
+    // Add ci label where applicable
+    {
+      "addLabels": [
+        "ci",
+      ],
+      "matchCategories": [
+        "ci",
+      ]
+    }
+  ],
+  
+  "pinDigest": true, // TODO: Check with https://docs.renovatebot.com/configuration-options/#pindigest
```

Comment:
@piontec While I went through this, I saw that this `true` value doesn't make sense here. Should this be `"pinDigests": true` (with an s at the end)?

https://docs.renovatebot.com/configuration-options/#pindigest
https://docs.renovatebot.com/configuration-options/#pindigests

Reply:
Yes, good catch!